src/ci/resources/enable_metadata_deleting.inc.conf

```
@@ -0,0 +1,17 @@
+# To use this partial config to activate deleting, include
+# it at the top of another config file (eg papi_v2_shared_application.inc.conf)
+
+# Note: DO NOT blindly copy this test file into a production instance. The value of 10 minutes before deletion is FAR TOO SHORT
+# for real situations!
+
+services.MetadataService {
+  config {
+    delete-metadata {
+      # How long to pause between deletion attempts which fail, or if there's nothing to delete:
+      backoff-interval = 10 seconds
+
+      # How long to require after workflow completion before going ahead with deletion:
+      delay-after-workflow-completion = 10 minutes
```

[optional] I prefer to avoid scary messages whenever possible

```suggestion
services.MetadataService {
  config {
    delete-metadata {
      # How long to pause between deletion attempts which fail, or if there's nothing to delete:
      backoff-interval = 10 seconds

      # How long to require after workflow completion before going ahead with deletion:
      delay-after-workflow-completion = 365 days
```